deployments/roman/config/common.yaml

```diff
@@ -35,9 +35,3 @@ jupyterhub:
     memory:
       guarantee: 8G
       limit: 15G
-  auth:
```

Comment:
can we put this in secrets instead?

Reply:
Yep,  that's what I did.  I removed it from common.yaml and added it to the secrets template.  It worked fine.  I removed it from common.yaml for both tess and roman.

Reply:
(It worked fine with real e-mails...)